src/toxcore/dht/server/mod.rs

```diff
@@ -289,29 +290,22 @@ impl Server {
         Box::new(future)
     }
 
-    fn is_kbucket_empty(&self) -> bool {
-        self.close_nodes.read().is_empty()
-    }
-
-    fn is_all_discarded(&self) -> bool {
-        self.close_nodes.read().is_all_discarded()
-    }
-
     /// Send NodesRequest to initial bootstapping nodes and nodes in Kbucket.
     fn send_bootstrap_and_ping(&self) -> IoFuture<()> {
-        // Send NodesRequest to bootstrap nodes
         let mut request_queue = self.request_queue.write();
-        let bootstrap_futures = self.initial_bootstrap.iter().cloned()
-            .map(|node| self.send_nodes_req(&mut request_queue, &node.into(), self.pk))
-            .collect::<Vec<_>>();
-
         let close_nodes = self.close_nodes.read();
-        let mut futures = close_nodes.iter()
-            .map(|node| self.send_nodes_req(&mut request_queue, &node, self.pk))
+        let futures = self.initial_bootstrap
+            .iter()
+            .cloned()
+            .chain(close_nodes
```

Comment:
better change the order of close_nodes with initial_bootstrap to make the code a little bit prettier

Reply:
Done.